basic/src/test/java/org/jboss/hal/testsuite/test/configuration/elytron/authentication/ElytronAuthenticationConfigurationTestCase.java

```diff
@@ -180,32 +185,33 @@ public void editAllowSASLMechanisms() throws Exception {
      */
     @Test
     public void toggleAnonymous() throws Exception {
-        final SnapshotBackup snapshotBackup = new SnapshotBackup();
-        client.apply(snapshotBackup.backup());
-
         final Address authenticationConfigurationAddress = createAuthenticationConfiguration();
         final boolean initialValue = false;
 
         try {
-            page.navigate();
-            page.switchToAuthenticationConfiguration()
-                    .getResourceManager()
-                    .selectByName(authenticationConfigurationAddress.getLastPairValue());
-            new ConfigChecker.Builder(client, authenticationConfigurationAddress)
-                    .configFragment(page.getConfigFragment())
-                    .editAndSave(ConfigChecker.InputType.CHECKBOX, ANONYMOUS, !initialValue)
-                    .verifyFormSaved()
-                    .verifyAttribute(ANONYMOUS, !initialValue);
-
-            Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
-
-            new ConfigChecker.Builder(client, authenticationConfigurationAddress)
-                    .configFragment(page.getConfigFragment())
-                    .editAndSave(ConfigChecker.InputType.CHECKBOX, ANONYMOUS, initialValue)
-                    .verifyFormSaved()
-                    .verifyAttribute(ANONYMOUS, initialValue);
+            elytronAuthenticationOperations.performActionOnAttributeAndRevertItsValueToOriginal(
+                    authenticationConfigurationAddress,
+                    () -> {
+                        page.navigate();
+                        page.switchToAuthenticationConfiguration()
+                                .getResourceManager()
+                                .selectByName(authenticationConfigurationAddress.getLastPairValue());
+                        new ConfigChecker.Builder(client, authenticationConfigurationAddress)
+                                .configFragment(page.getConfigFragment())
+                                .editAndSave(ConfigChecker.InputType.CHECKBOX, ANONYMOUS, !initialValue)
+                                .verifyFormSaved()
+                                .verifyAttribute(ANONYMOUS, !initialValue);
+
+                        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+
+                        new ConfigChecker.Builder(client, authenticationConfigurationAddress)
+                                .configFragment(page.getConfigFragment())
+                                .editAndSave(ConfigChecker.InputType.CHECKBOX, ANONYMOUS, initialValue)
+                                .verifyFormSaved()
+                                .verifyAttribute(ANONYMOUS, initialValue);                    },
```

Comment:
please remove the spaces before `}`. Ideally add new line before the `}` instead of the spaces.